src/components/Context/ContextProvider.js

```diff
@@ -4,24 +4,24 @@ import { ethers, utils, BigNumber } from 'ethers';
 import BeefyVault from '../../abis/BeefyVault.json';
 import ERC20 from '../../abis/ERC20.json';
 
-import vaults from '../../data/vaults';
 import addr from '../../data/addresses';
 
 import { fetchPrice } from '../../utils/fetchPrice';
 import { getHolders } from '../../utils/getHolders';
 import { getEarnings } from '../../utils/getEarnings';
 import { formatTvl } from '../../utils/format';
+import { getVaults } from "../../utils/getVaults";
 
 export const VaultsContext = createContext(null);
 
 const fetchVaultTvl = async ({ vault, signer }) => {
   try {
-    const vaultContract = new ethers.Contract(vault.contract, BeefyVault, signer);
+    const vaultContract = new ethers.Contract(vault.earnedTokenAddress, BeefyVault, signer);
     const vaultBalance = await vaultContract.balance();
   
     
     const price = await fetchPrice({ oracle: vault.oracle, id: vault.oracleId });
-    const normalizationFactor = 1000000000;
+    const normalizationFactor = 100000;

```

Comment:
Causes overflow in high prices LPs with low-decimals tokens, like HARD-BNB LP price is $39M
should be enough precision still as API returns ".00" scale